runtime/runtime-params-estimator/src/config.rs

```diff
@@ -44,4 +44,6 @@ pub struct Config {
     pub drop_os_cache: bool,
     /// Use in-memory test DB, useful to avoid variance caused by DB.
     pub in_memory_db: bool,
+    /// Only run a minimal check that's faster than trying to get accurate results.
+    pub check_only: bool,
```

Comment:
Nit: I generally find CLIs be neater when the flags provided mostly focus on *adding* functionality (even if enabled by default, they can be disabled with a `--no-*` or `--*=no/false/0` flag.) Among other things this makes it easier to then apply these flags to configure said functionality (e.g. if we eventually end up with multiple degrees to how heavily can the estimation run.)

In this case something like a `--full-estimation` or `--accurate` would make sense perhaps?